modules/openapi-generator/src/main/resources/JavaJaxRS/ApiException.mustache

```diff
@@ -1,10 +1,17 @@
 package {{apiPackage}};
 
 {{>generatedAnnotation}}
-public class ApiException extends Exception{
+public class ApiException extends Exception {
+
     private int code;
-    public ApiException (int code, String msg) {
+
+    public ApiException(int code, String msg) {
         super(msg);
         this.code = code;
     }
+
+    public int getCode() {
```

Comment:
Would be nice if you can add the proper javadoc as well?

Reply:
Sure thing!